### ISSUE TITLE
2025-02-14, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#83]
# Today's Arxiv Papers

(1) [MME-CoT: Benchmarking Chain-of-Thought in Large Multimodal Models for Reasoning Quality, Robustness, and Efficiency](http://arxiv.org/abs/2502.09621v1), 2025-02-13

(2) [Exploring the Potential of Encoder-free Architectures in 3D LMMs](http://arxiv.org/abs/2502.09620v1), 2025-02-13

(3) [Human-LLM Coevolution: Evidence from Academic Writing](http://arxiv.org/abs/2502.09606v1), 2025-02-13

(4) [SelfCite: Self-Supervised Alignment for Context Attribution in Large Language Models](http://arxiv.org/abs/2502.09604v1), 2025-02-13

(5) [Do LLMs Recognize Your Preferences? Evaluating Personalized Preference Following in LLMs](http://arxiv.org/abs/2502.09597v1), 2025-02-13

(6) [KIMAs: A Configurable Knowledge Integrated Multi-Agent System](http://arxiv.org/abs/2502.09596v1), 2025-02-13

(7) [Logical forms complement probability in understanding language model (and human) performance](http://arxiv.org/abs/2502.09589v1), 2025-02-13

(8) [Polymind: Parallel Visual Diagramming with Large Language Models to Support Prewriting Through Microtasks](http://arxiv.org/abs/2502.09577v1), 2025-02-13

(9) [Zero-shot generation of synthetic neurosurgical data with large language models](http://arxiv.org/abs/2502.09566v1), 2025-02-13

(10) [MDCrow: Automating Molecular Dynamics Workflows with Large Language Models](http://arxiv.org/abs/2502.09565v1), 2025-02-13

(11) [EmbodiedBench: Comprehensive Benchmarking Multi-modal Large Language Models for Vision-Driven Embodied Agents](http://arxiv.org/abs/2502.09560v1), 2025-02-13

(12) [Explainable AI-assisted Optimization for Feynman Integral Reduction](http://arxiv.org/abs/2502.09544v1), 2025-02-13

(13) [Mind the Gap! Choice Independence in Using Multilingual LLMs for Persuasive Co-Writing Tasks in Different Languages](http://arxiv.org/abs/2502.09532v1), 2025-02-13

(14) [Improve LLM-based Automatic Essay Scoring with Linguistic Features](http://arxiv.org/abs/2502.09497v1), 2025-02-13

(15) [Objective quantification of mood states using large language models](http://arxiv.org/abs/2502.09487v1), 2025-02-13

(16) [Pixel-Level Reasoning Segmentation via Multi-turn Conversations](http://arxiv.org/abs/2502.09447v1), 2025-02-13

(17) [On multi-token prediction for efficient LLM inference](http://arxiv.org/abs/2502.09419v1), 2025-02-13

(18) [SQuARE: Sequential Question Answering Reasoning Engine for Enhanced Chain-of-Thought in Large Language Models](http://arxiv.org/abs/2502.09390v1), 2025-02-13

(19) [Truth Knows No Language: Evaluating Truthfulness Beyond English](http://arxiv.org/abs/2502.09387v1), 2025-02-13

(20) [APT-LLM: Embedding-Based Anomaly Detection of Cyber Advanced Persistent Threats Using Large Language Models](http://arxiv.org/abs/2502.09385v1), 2025-02-13

(21) [Language Agents as Digital Representatives in Collective Decision-Making](http://arxiv.org/abs/2502.09369v1), 2025-02-13

(22) [ThunderServe: High-performance and Cost-efficient LLM Serving in Cloud Environments](http://arxiv.org/abs/2502.09334v1), 2025-02-13

(23) [Beyond English: The Impact of Prompt Translation Strategies across Languages and Tasks in Multilingual LLMs](http://arxiv.org/abs/2502.09331v1), 2025-02-13

(24) [Copilot Arena: A Platform for Code LLM Evaluation in the Wild](http://arxiv.org/abs/2502.09328v1), 2025-02-13

(25) [A Benchmark for Crime Surveillance Video Analysis with Large Models](http://arxiv.org/abs/2502.09325v1), 2025-02-13

(26) [A Judge-free LLM Open-ended Generation Benchmark Based on the Distributional Hypothesis](http://arxiv.org/abs/2502.09316v1), 2025-02-13

(27) [When the LM misunderstood the human chuckled: Analyzing garden path effects in humans and language models](http://arxiv.org/abs/2502.09307v1), 2025-02-13

(28) [KET-RAG: A Cost-Efficient Multi-Granular Indexing Framework for Graph-RAG](http://arxiv.org/abs/2502.09304v1), 2025-02-13

(29) [When do neural networks learn world models?](http://arxiv.org/abs/2502.09297v1), 2025-02-13

(30) [EmoAssist: Emotional Assistant for Visual Impairment Community](http://arxiv.org/abs/2502.09285v1), 2025-02-13

(31) [SparQLe: Speech Queries to Text Translation Through LLMs](http://arxiv.org/abs/2502.09284v1), 2025-02-13

(32) [From large language models to multimodal AI: A scoping review on the potential of generative AI in medicine](http://arxiv.org/abs/2502.09242v1), 2025-02-13

(33) [OpenBench: A New Benchmark and Baseline for Semantic Navigation in Smart Logistics](http://arxiv.org/abs/2502.09238v1), 2025-02-13

(34) [Reliable Conversational Agents under ASP Control that Understand Natural Language](http://dx.doi.org/10.4204/EPTCS.416.41), 2025-02-13

(35) [Data2Concept2Text: An Explainable Multilingual Framework for Data Analysis Narration](http://dx.doi.org/10.4204/EPTCS.416.13), 2025-02-13

(36) [LP-LM: No Hallucinations in Question Answering with Logic Programming](http://dx.doi.org/10.4204/EPTCS.416.5), 2025-02-13

(37) [Visual Graph Question Answering with ASP and LLMs for Language Parsing](http://dx.doi.org/10.4204/EPTCS.416.2), 2025-02-13

(38) [On LLM-generated Logic Programs and their Inference Execution Methods](http://dx.doi.org/10.4204/EPTCS.416.1), 2025-02-13

(39) [Logical Lease Litigation: Prolog and LLMs for Rental Law Compliance in New York](http://dx.doi.org/10.4204/EPTCS.416.4), 2025-02-13

(40) [Thinking beyond the anthropomorphic paradigm benefits LLM research](http://arxiv.org/abs/2502.09192v1), 2025-02-13

(41) [Matina: A Large-Scale 73B Token Persian Text Corpus](http://arxiv.org/abs/2502.09188v1), 2025-02-13

(42) [RefineCoder: Iterative Improving of Large Language Models via Adaptive Critique Refinement for Code Generation](http://arxiv.org/abs/2502.09183v1), 2025-02-13

(43) [FLAME: Flexible LLM-Assisted Moderation Engine](http://arxiv.org/abs/2502.09175v1), 2025-02-13

(44) [Two-Stage Representation Learning for Analyzing Movement Behavior Dynamics in People Living with Dementia](http://arxiv.org/abs/2502.09173v1), 2025-02-13

(45) [Improving TCM Question Answering through Tree-Organized Self-Reflective Retrieval with LLMs](http://arxiv.org/abs/2502.09156v1), 2025-02-13

(46) [LLM-Driven Augmented Reality Puppeteer: Controller-Free Voice-Commanded Robot Teleoperation](http://arxiv.org/abs/2502.09142v1), 2025-02-13

(47) [One-shot Federated Learning Methods: A Practical Guide](http://arxiv.org/abs/2502.09104v1), 2025-02-13

(48) [Bridging the Gap Between LLMs and Human Intentions: Progresses and Challenges in Instruction Understanding, Intention Reasoning, and Reliable Generation](http://arxiv.org/abs/2502.09101v1), 2025-02-13

(49) [Logical Reasoning in Large Language Models: A Survey](http://arxiv.org/abs/2502.09100v1), 2025-02-13

(50) [Show Me the Work: Fact-Checkers' Requirements for Explainable Automated Fact-Checking](http://dx.doi.org/10.1145/3706598.3713277), 2025-02-13

(51) [CoSER: Coordinating LLM-Based Persona Simulation of Established Roles](http://arxiv.org/abs/2502.09082v1), 2025-02-13

(52) [Enhancing RAG with Active Learning on Conversation Records: Reject Incapables and Answer Capables](http://arxiv.org/abs/2502.09073v1), 2025-02-13

(53) [CRANE: Reasoning with constrained LLM generation](http://arxiv.org/abs/2502.09061v1), 2025-02-13

(54) [Unleashing the Power of Large Language Model for Denoising Recommendation](http://dx.doi.org/10.1145/3696410.3714758), 2025-02-13

(55) [An Open Recipe: Adapting Language-Specific LLMs to a Reasoning Model in One Day via Model Merging](http://arxiv.org/abs/2502.09056v1), 2025-02-13

(56) [Cost-Saving LLM Cascades with Early Abstention](http://arxiv.org/abs/2502.09054v1), 2025-02-13

(57) [Game Theory Meets Large Language Models: A Systematic Survey](http://arxiv.org/abs/2502.09053v1), 2025-02-13

(58) [Typhoon T1: An Open Thai Reasoning Model](http://arxiv.org/abs/2502.09042v1), 2025-02-13

(59) [EventSTR: A Benchmark Dataset and Baselines for Event Stream based Scene Text Recognition](http://arxiv.org/abs/2502.09020v1), 2025-02-13

(60) [Diversity Enhances an LLM's Performance in RAG and Long-context Task](http://arxiv.org/abs/2502.09017v1), 2025-02-13

(61) [Hope vs. Hate: Understanding User Interactions with LGBTQ+ News Content in Mainstream US News Media through the Lens of Hope Speech](http://arxiv.org/abs/2502.09004v1), 2025-02-13

(62) [RoSTE: An Efficient Quantization-Aware Supervised Fine-Tuning Approach for Large Language Models](http://arxiv.org/abs/2502.09003v1), 2025-02-13

(63) [End-to-End triplet loss based fine-tuning for network embedding in effective PII detection](http://arxiv.org/abs/2502.09002v1), 2025-02-13

(64) [Task Generalization With AutoRegressive Compositional Structure: Can Learning From $\d$ Tasks Generalize to $\d^{T}$ Tasks?](http://arxiv.org/abs/2502.08991v1), 2025-02-13

(65) [Tuning-Free Personalized Alignment via Trial-Error-Explain In-Context Learning](http://arxiv.org/abs/2502.08972v1), 2025-02-13

(66) [RTBAS: Defending LLM Agents Against Prompt Injection and Privacy Leakage](http://arxiv.org/abs/2502.08966v1), 2025-02-13

(67) [Medicine on the Edge: Comparative Performance Analysis of On-Device LLMs for Clinical Reasoning](http://arxiv.org/abs/2502.08954v1), 2025-02-13

(68) [Structured Convergence in Large Language Model Representations via Hierarchical Latent Space Folding](http://arxiv.org/abs/2502.08947v1), 2025-02-13

(69) [The Stochastic Parrot on LLM's Shoulder: A Summative Assessment of Physical Concept Understanding](http://arxiv.org/abs/2502.08946v1), 2025-02-13

(70) [Beyond the Singular: The Essential Role of Multiple Generations in Effective Benchmark Evaluation and Analysis](http://arxiv.org/abs/2502.08943v1), 2025-02-13

(71) [Escaping Collapse: The Strength of Weak Data for Large Language Model Training](http://arxiv.org/abs/2502.08924v1), 2025-02-13

(72) [CopySpec: Accelerating LLMs with Speculative Copy-and-Paste Without Compromising Quality](http://arxiv.org/abs/2502.08923v1), 2025-02-13

(73) [Self-Consistency of the Internal Reward Models Improves Self-Rewarding Language Models](http://arxiv.org/abs/2502.08922v1), 2025-02-13

(74) [Exploring Emotion-Sensitive LLM-Based Conversational AI](http://arxiv.org/abs/2502.08920v1), 2025-02-13

(75) [InfiniteHiP: Extending Language Model Context Up to 3 Million Tokens on a Single GPU](http://arxiv.org/abs/2502.08910v1), 2025-02-13

(76) [Towards Automated Fact-Checking of Real-World Claims: Exploring Task Formulation and Assessment with LLMs](http://arxiv.org/abs/2502.08909v1), 2025-02-13

(77) [Reinforced Large Language Model is a formal theorem prover](http://arxiv.org/abs/2502.08908v1), 2025-02-13

(78) [DiffoRA: Enabling Parameter-Efficient LLM Fine-Tuning via Differential Low-Rank Matrix Adaptation](http://arxiv.org/abs/2502.08905v1), 2025-02-13

(79) [MIH-TCCT: Mitigating Inconsistent Hallucinations in LLMs via Event-Driven Text-Code Cyclic Training](http://arxiv.org/abs/2502.08904v1), 2025-02-13

(80) [3D-Grounded Vision-Language Framework for Robotic Task Planning: Automated Prompt Synthesis and Supervised Reasoning](http://arxiv.org/abs/2502.08903v1), 2025-02-13

(81) [Communication is All You Need: Persuasion Dataset Construction via Multi-LLM Communication](http://arxiv.org/abs/2502.08896v1), 2025-02-13

(82) [LLM-Enhanced Multiple Instance Learning for Joint Rumor and Stance Detection with Social Context Information](http://arxiv.org/abs/2502.08888v1), 2025-02-13

(83) [ShapeLib: designing a library of procedural 3D shape abstractions with Large Language Models](http://arxiv.org/abs/2502.08884v1), 2025-02-13

(84) [Utilizing Pre-trained and Large Language Models for 10-K Items Segmentation](http://arxiv.org/abs/2502.08875v1), 2025-02-13

(85) [Harnessing Vision Models for Time Series Analysis: A Survey](http://arxiv.org/abs/2502.08869v1), 2025-02-13

(86) [Ask in Any Modality: A Comprehensive Survey on Multimodal Retrieval-Augmented Generation](http://arxiv.org/abs/2502.08826v1), 2025-02-12

(87) [Can a Single Model Master Both Multi-turn Conversations and Tool Use? CALM: A Unified Conversational Agentic Language Model](http://arxiv.org/abs/2502.08820v1), 2025-02-12

(88) [Lexical Manifold Reconfiguration in Large Language Models: A Novel Architectural Approach for Contextual Modulation](http://arxiv.org/abs/2502.08818v1), 2025-02-12

(89) [A Systematic Review on the Evaluation of Large Language Models in Theory of Mind Tasks](http://arxiv.org/abs/2502.08796v1), 2025-02-12

(90) [Spectral Journey: How Transformers Predict the Shortest Path](http://arxiv.org/abs/2502.08794v1), 2025-02-12

(91) [If Multi-Agent Debate is the Answer, What is the Question?](http://arxiv.org/abs/2502.08788v1), 2025-02-12

(92) [Zero-Shot Belief: A Hard Problem for LLMs](http://arxiv.org/abs/2502.08777v1), 2025-02-12

(93) [Universal Model Routing for Efficient LLM Inference](http://arxiv.org/abs/2502.08773v1), 2025-02-12

(94) [From PowerPoint UI Sketches to Web-Based Applications: Pattern-Driven Code Generation for GIS Dashboard Development Using Knowledge-Augmented LLMs, Context-Aware Visual Prompting, and the React Framework](http://arxiv.org/abs/2502.08756v1), 2025-02-12

(95) [Beyond the Lens: Quantifying the Impact of Scientific Documentaries through Amazon Reviews](http://arxiv.org/abs/2502.08705v1), 2025-02-12

(96) [Utility Engineering: Analyzing and Controlling Emergent Value Systems in AIs](http://arxiv.org/abs/2502.08640v1), 2025-02-12

(97) [Examining Multilingual Embedding Models Cross-Lingually Through LLM-Generated Adversarial Examples](http://arxiv.org/abs/2502.08638v1), 2025-02-12

(98) [Ensemble based approach to quantifying uncertainty of LLM based classifications](http://arxiv.org/abs/2502.08631v1), 2025-02-12

(99) [SPeCtrum: A Grounded Framework for Multidimensional Identity Representation in LLM-Based Agent](http://arxiv.org/abs/2502.08599v1), 2025-02-12

(100) [Commercial LLM Agents Are Already Vulnerable to Simple Yet Dangerous Attacks](http://arxiv.org/abs/2502.08586v1), 2025-02-12
